spec/server/spec_helper.js

```diff
@@ -0,0 +1,20 @@
+require('../support/bluebird');
+require('../spec_helper');
+const {tables} = require('../../server/db.json');
```

Comment:
do we need destructuring here? Not a big deal but it's a bit easier to read, would change line 12 to `await db.init(tables, conn);`